### ISSUE TITLE
URGENT: Unable to display edited content - old, cached content is displayed to users

### ISSUE THREAD

**Issue by berthelemy** [andrewhancox/moodle-filter_translations#129]
## To replicate

1. Switch on the content translation filter
2. Create a quiz question with an image with the wrong image url, and wrong ALT text
3. Save and preview the question
4. Edit the question to fix the url and replace the ALT text
5. Save and preview the question

### Expected behaviour

At (3) - the question should display with a missing image, and the entered ALT text displayed.

![image](https://user-images.githubusercontent.com/2512079/175946781-970fcbfb-310a-4169-9d19-16d2e066b742.png)

At (5) - the question should display with the correct image, and the correct ALT text displayed.

![image](https://user-images.githubusercontent.com/2512079/175946907-feb9d230-0337-4307-a057-233e70a55066.png)

### Actual behaviour

At (3)  - the question displays with a missing image, and the entered (incorrect) ALT text displayed. (As in the first screenshot)

At (5) -  - the question displays with a missing image, and the entered (incorrect) ALT text displayed. (As in the first screenshot)

### Actual behaviour

At (3)  - the question displays with a missing image, and the entered (incorrect) ALT text displayed. (As in the first screenshot)

At (5) -  the question displays with the correct image, and the correct ALT text displayed. (As in the second screenshot)

**Comment by berthelemy** [comment]
It's not just the quizzes. It's all content. Changes made are not being reflected in the content displayed to the user. Instead we only see the cached version.

Turning off the content translation filter allows the "real" content to be displayed.

The cached version should be deleted if the content has been edited? Shouldn't it?

**Comment by berthelemy** [comment]
@jgramp Are you seeing this issue on the Moodle Academy? We're using the most recent version, from last week.

**Comment by andrewhancox** [comment]
Just trying to figure out an approach to resolving this...

**Comment by andrewhancox** [comment]
This will only be an issue on auto-translated content so won't be impacting the academy.

**Comment by andrewhancox** [comment]
@berthelemy  I've pushed a commit - the system will now only use stale translations that were created manually - this should fix your issue.

**Comment by berthelemy** [comment]
Thanks. This seems to be fixed.
